src/com/company/Main.java

```diff
@@ -35,66 +34,62 @@ public static void play() {
         int rowNumber = 0;
         int columnNumber = 0;
 
-        shouldPlayAgain = true;
+        PlayAgain = true;
 
-        boardSize = getPlayingFieldSize(MAX_BOARD_SIZE);
+        final int boardSize = getPlayingFieldSize(MAX_BOARD_SIZE);
         System.out.println(SELECTED + boardSize);
 
-        final PlayerSignType[][] board = new PlayerSignType[boardSize][boardSize];
-
-        for (int i = 0; i < boardSize; i++) {
-            for (int j = 0; j < boardSize; j++) {
-                board[i][j] = PlayerSignType.EMPTY;
-            }
-        }
+//        final PlayerSignType[][] board = new PlayerSignType[boardSize][boardSize];
+        final Board board = new Board(boardSize);
 
         drawField(boardSize, board);
         do {
             rowNumber = getRowNumber(boardSize);
             columnNumber = getColumnNumber(boardSize);
             System.out.println(SELECTED + rowNumber + "/" + columnNumber);  //czy to zmieniać - jest numer powinna być litera
-            if (!putToTheBoard(rowNumber, columnNumber, X, board)) {
+            if (!putToBoard(rowNumber, columnNumber, X, board)) {
                 continue;
             }
 
             drawField(boardSize, board);
             //sprawdzenie
-            shouldPlayAgain = verifyIfContinue(boardSize, board);
+            PlayAgain = verifyIfContinue(boardSize, board);
 
             //ruch "O"
             //drawField(boardSize,board);
 
 
-        } while (shouldPlayAgain);
+        } while (PlayAgain);
         System.out.println(END_OF_THE_GAME);
     }
 
-    private static boolean putToTheBoard(final int rowNumber, final int columnNumber, final PlayerSignType oneSing, final PlayerSignType[][] board) {
-        if (board[rowNumber - 1][columnNumber - 1] == PlayerSignType.EMPTY) {
-            board[rowNumber - 1][columnNumber - 1] = oneSing;
-        } else {
+    private static boolean putToBoard(final int row, final int column, final PlayerSignType Sing, Board board) {
+        if (!board.insertSign(Sing, row - 1, column - 1)) {
             System.out.println(NOT_EMPTY_PLACE);
             return false;
         }
         return true;
     }
 
-    private static boolean verifyIfContinue(final int boardSize, final PlayerSignType[][] board) {
+    private static boolean verifyIfContinue(final int boardSize, Board board) {
         return verifyIfRowIsNotFull(boardSize, board)
                 && verifyIfColumnIsNotFull(boardSize, board)
                 && verifyIfDiagonalXxIsNotFull(boardSize, board)
                 && verifyIfDiagonalYyIsNotFull(boardSize, board);
     }
 
-    public static boolean verifyIfDiagonalYyIsNotFull(final int boardSize, final PlayerSignType[][] board) {
+    public static boolean verifyIfDiagonalYyIsNotFull(final int boardSize, Board board) {
```

Comment:
final (czemu je pousuwałaś?)

Przy okazji czemu tych metod nie przeniosłaś do klasy Board? przecież to ona powinna być odpowiedzialna za policzenie tego co ma w środku.